content/show/coder-radio/_index.md

```diff
@@ -6,6 +6,8 @@ draft = false
 categories = ["Coder Radio"]
 show = "coder-radio"
 hosts = ["chris","michael", "wes"]
+live = "Mondays<br>9:00am Pacific<br>12:00pm Eastern"
```

Comment:
I definitely don't plan to have HTML here.  Instead it will be some parse-able value we can use to display the time based on the chosen timezone.